### PR TITLE
Fixing organs not burning in fires.

### PR DESCRIPTION
**Pull request by MistakeNot4892**
Organs will now burn to ash on a fire.

### CODE REVIEW

**Comment by MistakeNot4892** [comment]
Temperature damage is stupid.

File: code/modules/materials/_material_stack.dm [hunk 1]
```diff
@@ -219,3 +219,8 @@
 		. += " [singular_name]"
 		return indefinite_article ? "[indefinite_article] [.]" : ADD_ARTICLE(.)
 	return "[amount] [.] [plural_name]"
+
+// Horrible solution to heat damage for atoms causing logs and
+// fuel to vanish. Replace this when the atom fire system exists.
+/obj/item/stack/material/should_take_heat_damage(datum/gas_mixture/air, exposed_temperature)
+	return istype(loc, /obj/structure/fire_source) ? FALSE : ..()
```
Comment (out-of-phaze) [inline code/modules/materials/_material_stack.dm:226]: does nothing to stop molds taking damage when you fire them in a kiln

**Review by out-of-phaze — CHANGES_REQUESTED** [review]

File: code/modules/materials/_material_stack.dm (hunk 1 above)
Comment (MistakeNot4892) [reply code/modules/materials/_material_stack.dm:226]: Molds generally don't have an ignition temperature, these changes shouldn't impact them.

Comment (MistakeNot4892) [reply code/modules/materials/_material_stack.dm:226]: Per Discord, I'm not sure what change is being requested here. I haven't adjusted the behavior for materials like clay that don't solely have an ignition point.

**Review by out-of-phaze — APPROVED** [review]
as i explained on discord i misunderstood

**Pull request merged**